app/treatmentcenterMap/map/index.js

```diff
@@ -0,0 +1,56 @@
+var config = require('./map.json');
```

Comment:
There's no need to copy map component here. You can add an interface in the original map component to accomplish your requirement. (like adding a callback function when you want to change the url.)